src/editor/nodes/ModelNode.js

```diff
@@ -115,15 +116,7 @@ export default class ModelNode extends EditorNodeMixin(Model) {
 
     const clonedScene = cloneObject3D(scene);
 
-    const sketchfabExtras = json.asset && json.asset.extras;
-
-    if (sketchfabExtras) {
-      const name = sketchfabExtras.title;
-      const author = sketchfabExtras.author ? sketchfabExtras.author.replace(/ \(http.+\)/, "") : "";
```

Comment:
We may want to keep this string replacement to remove the author url from the author field. Otherwise you get an attribution that looks like this: 
<img width="369" alt="Screen Shot 2021-01-12 at 4 35 46 PM" src="https://user-images.githubusercontent.com/1753624/104391212-474e4f80-54f4-11eb-901c-6f6464aaf5bd.png">

Reply:
Fixed